numpy/random/bit_generator.pyi

```diff
@@ -0,0 +1,89 @@
+import abc
+from threading import Lock
+from typing import (Any, Callable, Dict, List, Literal, NamedTuple, Optional,
+                    Sequence, Tuple, Union, overload)
+
+from numpy import dtype, ndarray, uint32, uint64
+from numpy.typing import DTypeLike, _ArrayLikeInt_co, _ShapeLike
+
+class _Interface(NamedTuple):
+    state_address: Any
+    state: Any
+    next_uint64: Any
+    next_uint32: Any
+    next_double: Any
+    bit_generator: Any
+
+class ISeedSequence(abc.ABC):
+    @abc.abstractmethod
+    def generate_state(
+        self, n_words: int, dtype: DTypeLike = ...
+    ) -> ndarray[Any, Any]: ...
+
+class ISpawnableSeedSequence(ISeedSequence):
+    @abc.abstractmethod
+    def spawn(self, n_children: int) -> List[ISpawnableSeedSequence]: ...
```

Comment:
```suggestion
_T = TypeVar("_T")

class ISpawnableSeedSequence(ISeedSequence):
    @abc.abstractmethod
    def spawn(self: _T, n_children: int) -> List[_T]: ...
```
Using a typevar both here and in `SeedlessSeedSequence` should resolve the first mypy complaint.